.setup/bin/reupload_old_assignments.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3 
+import sys
+import os
+import json
+from sqlalchemy import create_engine, Table, MetaData, bindparam, and_
+import shutil
+
+SUBMITTY_DATA_DIR = "/var/local/submitty"
+
+sys.path.append("/usr/local/submitty/bin")
+import submitty_utils
+
+DB_HOST = "localhost"
+DB_USER = "hsdbu"
+DB_PASS = "hsdbu"
+
+
+
+# This script was created to help professors re-upload an old semester's assignments for autograding.
+# To run it, please create a new assignment (hereafter called the CURRENT assignment) via the submitty interface 
+# to which you wish to upload your old assignment. Then, run this script and provide the following arguments:
+# ARGUMENTS
+# 1) The path to the top level of the old assignment's directory tree. This folder's subdirectories should mirror 
+#    a /var/local/submitty/courses/<semester>/<course>/submissions/<assignment_name> folder.
+# 2) The semester of the course to which the old assignments will be uploaded.
+# 3) The name of the course to which the old assignments will be uploaded.
+# 4) The name of the current assignment to which the old assignments will be uploaded
+# 5) The optional argument --grade, which causes the uploaded assignments to be added to the grading queue.
+
+#NOTE: variables beginning with ARCHIVED_ are populated using the homework assignment being copied.
+#Varialbes beginning with CURRENT_ are populated using the destination/current semester's directories.
+def main():
+    grade = False
+    #In order to successfully re-upload the assignment, we need the directory of the assignment to be inflated,
+    #and the semester, course name, and assignment name that it should be uploaded to.
+    semester = course_name = assignment_name = ARCHIVED_directory =""
+    if len(sys.argv) < 5:
+        print("ERROR: Please provide:\n\t1)Directory with your files\n\t2)semester \n\t3)course code \n\t4)assignment name.\n\t5)--grade (optional)")
+        sys.exit(1)
+    else:
+        ARCHIVED_directory = sys.argv[1]
+        if not os.path.isdir(ARCHIVED_directory):
+            print("The provided directory (" + ARCHIVED_directory + + ") does not exist.")
+            sys.exit(1)
+        semester = sys.argv[2]
+        course_name = sys.argv[3]
+        assignment_name = sys.argv[4]
+        #The optional argument --grade causes the uploaded assignment to be added to the grading queue.
+        if len(sys.argv) == 6:
+            if sys.argv[5] == "--grade":
+                grade = True
```

Comment:
Use [argparse](https://docs.python.org/3/library/argparse.html) instead of `sys.argv`.